src/Models/UserModel.php

```diff
@@ -14,6 +14,9 @@
 use CodeIgniter\Shield\Exceptions\ValidationException;
 use Faker\Generator;
 
+/**
+ * @psalm-consistent-constructor
```

Comment:
Might as well tell PHPStan as well:
```suggestion
 * @phpstan-consistent-constructor
 * @psalm-consistent-constructor
```

https://phpstan.org/blog/solving-phpstan-error-unsafe-usage-of-new-static#add-%40phpstan-consistent-constructor-to-the-class

Reply:
It is not true to all tags, but PHPStan vendor tags are often supported by psalm as well.
So I changed the tag to PHPStan's.

I don't know why but the issues are reduced to 681.